src/ux/step1/WalletProviders.tsx

```diff
@@ -47,11 +69,23 @@ export const WalletProviders = ({ userProviders, setLoading }: IWalletProvidersP
       ) : null
     )}
   </ProvidersWrapper>
-  <Paragraph className={PROVIDERS_FOOTER_TEXT_CLASSNAME}>
 
-    <Trans>No wallet? </Trans>
-    <NoWalletAnchor href="https://developers.rsk.co/wallet/use/" target="_blank" className={ANCHOR_CLASSNAME}>
-      <Trans>Get one here!</Trans>
-    </NoWalletAnchor>
-  </Paragraph>
+  <FooterWrapper >
+    <Paragraph>
+      <LanguageSelector onChange={(val) => changeLanguage(val.target.value)} defaultValue={selectedLanguageCode} name="languages" id="languages">
```

Comment:
```suggestion
      <LanguageSelector onChange={(e) => changeLanguage(e.target.value)} defaultValue={selectedLanguageCode} name="languages" id="languages">
```

More declarative, for`event`